packages/sidebar/src/Sidebar.tsx

```diff
@@ -21,116 +23,105 @@ import {
   Separator,
   CloseOpenButton,
   AntiSelect,
-  Scrollbar,
-} from './styled/Sidebar';
+} from './styled';
 
 
-export const PureSidebar = ({
+export const PureSidebar: React.FC<ISidebarProps> = ({
   theme,
   children,
   navComponent,
   showScrollbar,
   arrowComponent = <Arrow />,
+  isResponsive = false,
   withArrow = true,
   isRTL = false,
   minWidth = 30,
   maxWidth = 400,
-  separatorWidth: separatorWidthProp = 90,
-}: ISidebarProps): ReactElement => {
+  navWidth = 90,
+  separatorWidth = 10,
+}) => {
   const themeContext = useContext(ThemeContext);
 
-  const [transformParams, setTransformParams] = useState({
-    width: maxWidth,
-    animate: false,
-    arrowToRight: false,
-  });
-
-  const [antiSelectLayer, changeAntiSelectLayer] = useState(false);
-  const sidebarRef = useRef<HTMLDivElement>(null);
-
-  const clickX = useRef(0);
+  const [width, setWidth] = useState(maxWidth);
```

Comment:
You made some state and you use them together. I think pointless refactoring :) What you think about it?

Reply:
1) It is not obvious what is it transformParams, and it is hard and wrong to combine incombineble.
2) With current changes it is pretty clear what must be changed in concrete case.